include/sharg/parser.hpp

```diff
@@ -747,6 +747,14 @@ class parser
      */
     void init(int argc, char const * const * const argv)
     {
+        info.executable_name.emplace_back(argv[0]);
+
+        if (argc <= 1) // no arguments provided
+        {
+            format = detail::format_short_help{};
+            return;
+        }
+
```

Comment:
Why do you need this here?

Reply:
I noticed it too, and talked to Simon in person. It's an artifact from rebasing, has to be removed.